### ISSUE TITLE
Unable to run phrase if another user has already ran it

### ISSUE THREAD

**Issue by jdeniau** [phrase/phrase-cli#86]
On a computer with multiple user, if the user `a` run a phrase command, and then a user `b` run a phrase command, the user `b` will have the following error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8f46f3]

goroutine 1 [running]:
github.com/phrase/phrase-cli/cmd/internal/updatechecker.(*Checker).Check(0xc000593d30)
	/go/src/github.com/phrase/phrase-cli/cmd/internal/updatechecker/update_checker.go:54 +0x2b3
github.com/phrase/phrase-cli/cmd.checkUpdate()
	/go/src/github.com/phrase/phrase-cli/cmd/root.go:180 +0x136
github.com/spf13/cobra.(*Command).preRun(0xc0002a1b80)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:872 +0x49
github.com/spf13/cobra.(*Command).execute(0xc0002a1b80, 0xe82600, 0x0, 0x0, 0xc0002a1b80, 0xe82600)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:808 +0x14f
github.com/spf13/cobra.(*Command).ExecuteC(0xe49740, 0x41aa01, 0x0, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/phrase/phrase-cli/cmd.Execute()
	/go/src/github.com/phrase/phrase-cli/cmd/root.go:64 +0x2d
main.main()
	/go/src/github.com/phrase/phrase-cli/main.go:8 +0x25
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

After debugging, it comes from the [`/tmp/.phrase.version`](https://github.com/phrase/phrase-cli/blob/b2e203660dd8de5019ad6a7730e001d3f0fa4240/cmd/root.go#L175) file, which is written by the user `a` with the following access : 700.

Changing access to `777`  with `chmod 777 /tmp/.phrase.version` does fix the problem, but it's obviously a temporary solution.

**Comment by vakuum** [comment]
I have the same problem after switching from https://github.com/phrase/phraseapp-client to https://github.com/phrase/phrase-cli.

```
$ id -un
aaaaa

$ strace -f phrase version
...
[pid 2856004] openat(AT_FDCWD, "/tmp/.phrase.version", O_WRONLY|O_CREAT|O_TRUNC|O_CLOEXEC, 0600 <unfinished ...>
...
[pid 2858836] --- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=NULL} ---
...

$ ls -la /tmp/.phrase.version
-rw------- 1 bbbbb bbbbb 5 Sep  2 17:25 /tmp/.phrase.version
```

**Comment by theSoenke** [comment]
Hi, thanks for reporting this issue. We will fix this in the new CLI phrase-cli

**Comment by theSoenke** [comment]
This should now be resolved with phrase-cli v2.2.1. The file is now written with `666`
